src/main/features/core/lyrics/source_musiXmatch.js

```diff
@@ -0,0 +1,42 @@
+import { AllHtmlEntities as Entities } from 'html-entities';
+import fetch from 'node-fetch';
+import xss from 'xss';
+import JSON from 'JSON';
+
+const decoder = new Entities();
+
+const attemptMusiXmatch = (path) => {
+  return new Promise((resolve, reject) => {
+    fetch(`https://www.musixmatch.com/search/${path}`) // the lyrics urls aren't always very straightforward...
+      .then((data) => data.text())
+      .then((searchHtml) => {
+        const searchReturn = JSON.parse(/__mxmProps = ({.+})<\/script>/.exec(searchHtml)[1]);
+        fetch(searchReturn.allResults.bestMatch.shareURI)
```

Comment:
Seems a bit dodgy to go down the "it didn't work let the promise go to the catch method" route.  It seems akin to wrapping your entire program in a `try { ... } catch` block.  Although in this case it works and is semi appropriate, it seems bad practice in general.